app/models/hearing.js

```diff
@@ -0,0 +1,20 @@
+import DS from 'ember-data';
+import { attr, belongsTo } from '@ember-decorators/data';
+
+const { Model } = DS;
+
+export default class HearingModel extends Model {
+// DB Table: dcp_communityboarddisposition
+
+  // One Action to One Hearing
+  @belongsTo('action') action;
```

Comment:
Hearing belongs to project, not action. LUPs only schedule one hearing for a project, regardless of how many actions it has.

Reply:
Okay cool so the relationship is: ONE Project to ONE Hearing--gotcha!